components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -774,8 +774,18 @@ def _get_prepared_image(request, iid, server_id=None, conn=None,
         return
     if 'c' in r:
         logger.debug("c="+r['c'])
-        channels, windows, colors, reverses = _split_channel_info(r['c'])
-        if not img.setActiveChannels(channels, windows, colors, reverses):
+        activechannels, windows, colors, reverses = _split_channel_info(r['c'])
+        allchannels = []
```

Comment:
@will-moore This code will have to be reviewed in https://github.com/openmicroscopy/openmicroscopy/pull/5082
Merging so you can move forward